src/methods/method_overloading/hard/q5/Averages.java

```diff
@@ -0,0 +1,57 @@
+package methods.method_overloading.hard.q5;
+
+/*
+Write a Java program to calculate the average of sets of 3, 4, and 5 numbers. To do this, create a method called average
+and overload it to accept 3, 4, and integers, and then returns the average of the set of numbers as a double. Your
+program should prompt the user to enter a set of 3, 4, and 5 numbers, calculate the average for each set and print out
+the average in the main method.
+
+Sample Output:
+  Enter three numbers:
+  2 1 4
+  Enter four numbers:
+  6 31 3 51
+  Enter five numbers:
+  2 1 4 5 999
+  The average of the following numbers is: 2.3333333333333335
```

Comment:
Average of the following doesn't make sense. You should say first set, second set, etc.